app/views/govuk_publishing_components/components/_contextual_sidebar.html.erb

```diff
@@ -1,12 +1,15 @@
-<% navigation = GovukPublishingComponents::Presenters::ContextualNavigation.new(content_item, request.path) %>
+<%
+  active_step_nav_content_id = request.query_parameters['step-by-step-nav']
+  navigation = GovukPublishingComponents::Presenters::ContextualNavigation.new(content_item, request.path, active_step_nav_content_id) 
```

Comment:
Perhaps change `GovukPublishingComponents::Presenters::ContextualNavigation` to accept the whole `request` object as a parameter, rather than adding `active_step_nav_content_id` as a new one.